index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mahmoud Makhlouf</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <div class="wraper">
+
+        <nav class="header">
+            <h1>Mahmoud Makhlouf</h1>
+            <a href="#">
+                <img src="Union.png" alt="collapsed-menu">
+            </a>
+        </nav>
+
+        <div class="container">
+
+            <div class="headline">
+                <h1>Hey There. I’m Mahmoud</h1>
+                <h3>I’am a Software Developer</h3>
+                <p>I can help you build a product ,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.
+                </p>
+                <div class="icon-bar">
+                    <a href="#">
+                        <img src="github.png" alt="github-icon">
+                    </a>
+                    <a href="#">
+                        <img src="in.png" alt="linkedin-icon">
+                    </a>
+                    <a href="#">
+                        <img src="Vector.png" alt="victory-sign">
+                    </a>
+                    <a href="#">
+                        <img src="twitter.png" alt="twitter-icon">
+                    </a>
+                    <a href="#">
+                        <img src="m.png" alt="m-letter">
+                    </a>
```

Comment:
- [ ] Please do your possible best to wrap the social media icons in `ul tag` and each icon should be inside a `li tag`.